tree/ntuple/v7/test/rfield_vector.cxx

```diff
@@ -83,13 +83,22 @@ TEST(RNTuple, InsideCollection)
    field->SetOnDiskId(idKlassVec);
    field->ConnectPageSource(*source);
 
+   auto fieldCardinality =
+      std::unique_ptr<RFieldBase>(RFieldBase::Create("", "ROOT::Experimental::RNTupleCardinality").Unwrap());
+   fieldCardinality->SetOnDiskId(idKlassVec);
+   fieldCardinality->ConnectPageSource(*source);
+
    auto value = field->GenerateValue();
    field->Read(0, &value);
    auto aVec = value.Get<std::vector<float>>();
    EXPECT_EQ(1U, aVec->size());
    EXPECT_EQ(42.0, (*aVec)[0]);
    field->DestroyValue(value);
 
+   auto valueCardinality = fieldCardinality->GenerateValue();
```

Comment:
This overload of `GenerateValue()` uses `malloc()`, no?  Either a `DestroyValue()` is missing, or maybe replace by `GenerateValue(void *where)`?